### PR TITLE
anydesk: 6.0.1 -> 6.1.0

### PR DESCRIPTION
**Pull request by otavio**
###### Motivation for this change

New release.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/remote/anydesk/default.nix [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 let
   sha256 = {
-    x86_64-linux = "19751ygq1ng79aniqx91qawc0cw07cwdjdjd88azc9ww6z6nv0mp";
+    x86_64-linux = "1qbq6r0yanjappsi8yglw8r54bwf32bjb2i63awmr6pk5kmhhy3r";
     i386-linux   = "0dwc7v4p1dz51444zwn0kds23yi87r4h2d3isfj9xwkn90pxb7in";
```
Comment (shyim) [inline pkgs/applications/networking/remote/anydesk/default.nix:9]: 6.1.0 has dropped support for 32Bit. Could you remove it here too?

Comment (otavio) [reply pkgs/applications/networking/remote/anydesk/default.nix]: Done.

**Review by shyim — APPROVED** [review]

**Comment by ajs124** [comment]
We should probably either keep the `throw` around or adjust the `meta.platforms` attribute, so this throws an error, if anyone tries to build it on anything but x86_64 linux.

**Comment by otavio** [comment]
Good point. I think adjusting the `meta.platforms` seems cleaner to me. What do you think?

**Comment by otavio** [comment]
Gentle ping, asking for a review.

**Pull request merged**